src/gf-mksquashfs

```diff
@@ -30,9 +30,12 @@ set -x
 # https://github.com/coreos/coreos-assembler/pull/394
 tmpd=$(mktemp -tdp "$(dirname "${dest}")" gf-mksquashfs.XXXXXX)
 tmp_dest=${tmpd}/image.squashfs
-
 coreos_gf_run_mount "${src}" --ro
 
+# Remove the sysroot=readonly flag, see https://github.com/coreos/fedora-coreos-tracker/issues/589
+coreos_gf download /ostree/repo/config "${tmpd}/config"
+grep -v readonly=true "${tmpd}/config" > "${tmpd}/config.new"
+coreos_gf upload "${tmpd}/config.new" /ostree/repo/config
```

Comment:
WDYT about

```suggestion
coreos_gf debug sh "ostree config unset sysroot.readonly --repo /ostree/repo"
```

?

`debug` is technically not public, but OTOH we also already make use of it a lot, so that ship has already sailed. :)

Reply:
Mmm...I tested this one, let's ship it?

Longer term I think what we want to do is replace all the libguestfs usage overall with "boot the qemu image we just built" instead.

Reply:
OK I tried that and it doesn't work
```
+ guestfish --remote -- debug sh 'ostree config unset sysroot.readonly --repo /ostree/repo'
libguestfs: error: debug: /bin/sh: ostree: command not found
```

Because yeah ostree isn't in the libguestfs image - circling back to we should get away from libguestfs in general per above.